app/tests/v2/test.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+import json
+from ....app import launcher
+
+class TestEndPointsWithDatabases(unittest.TestCase):
+    def setUp(self):
+        self.app = launcher(app_config["development"])
+        self.client = self.app.test_client
+        #user_id should be autogenerated + date registered
```

Comment:
When you write a comment, you should start with a blank space (PEP8)
```suggestion
        # user_id should be autogenerated + date registered
```